site/content/en/docs/Guides/metrics.md

```diff
@@ -76,38 +75,6 @@ Follow the [Stackdriver Installation steps](#stackdriver-installation) to see yo
 | agones_k8s_client_workqueue_retries_total         | Total number of items retried to the work queue                          | counter   |
 | agones_k8s_client_workqueue_longest_running_processor         | How long the longest running workqueue processor has been running in microseconds  | gauge   |
```

Comment:
`| agones_gameservers_state_duration  | The distribution of gameserver state duration in seconds. Note: this metric could have some missing samples by design. Do not use the `_total` counter as the real value for state changes.     | histogram     |`

Was in the wrong block, so it looks like I delete the wrong one, but this made sure it stayed for the release.